src/modals/import-account-modal/import-account-private-key-form/import-account-private-key-form.tsx

```diff
@@ -0,0 +1,61 @@
+import { Formik } from 'formik';
+import React, { Dispatch, FC, SetStateAction } from 'react';
+import { View } from 'react-native';
+
+import { ButtonLargePrimary } from '../../../components/button/button-large/button-large-primary/button-large-primary';
+import { ButtonLargeSecondary } from '../../../components/button/button-large/button-large-secondary/button-large-secondary';
+import { ButtonsContainer } from '../../../components/button/buttons-container/buttons-container';
+import { Divider } from '../../../components/divider/divider';
+import { InsetSubstitute } from '../../../components/inset-substitute/inset-substitute';
+import { Label } from '../../../components/label/label';
+import { ScreenContainer } from '../../../components/screen-container/screen-container';
+import { FormMnemonicInput } from '../../../form/form-mnemonic-input';
+import { ImportAccountPrivateKeyValues } from '../../../interfaces/import-account-type';
+import { useShelter } from '../../../shelter/use-shelter.hook';
+import { useAccountsListSelector } from '../../../store/wallet/wallet-selectors';
+import { formatSize } from '../../../styles/format-size';
+import {
+  importAccountPrivateKeyFormInitialValues,
+  importAccountPrivateKeyFormValidationSchema
+} from '../import-account-modal.form';
+
+interface Props {
+  importAccountStep: number;
+  setImportAccountStep: Dispatch<SetStateAction<number>>;
```

Comment:
do not pass this prop into child component (return submit/press handlers). 
take a look at `AddTokenModal` as example